### ISSUE TITLE
Error thrown when trying to use ftp.site()

### ISSUE THREAD

**Issue by Sk1Zy** [realtymaps/promise-ftp#3]
Was trying to active UTF8 mode on microsoft IIS ftp server using site command.

My code:
```
let ftp = new PromiseFTP();

        return ftp
            .connect({
                host: config.ftp.host,
                user: config.ftp.user,
                password: config.ftp.password
            })
            .then(message => {
                return ftp.site('OPTS UTF8 ON');
            })
            .then(() => {
                return Promise.all(_.map(files, file => {
                        return ftp.put(file.buffer, file.originalname)
                    }))
                    .then(() => {
                        return ftp.end();
                    })
            })
```

Here's the error that gets thrown:
```
TypeError: promisifiedClientMethods.site(...).spread is not a function
    at PromiseFtp.site (/node_modules/promise-ftp/dist/promiseFtp.js:155:55)
    at /node_modules/promise-ftp/dist/promiseFtp.js:223:28
    at process._tickCallback (node.js:368:9)
```

I'm not sure whether i'm supposed to use the site command to achieve this but this error seemed to me to be unrelated to what i was trying to do.

**Comment by zacronos** [comment]
Right you are -- `.spread()` is a [bluebird](https://github.com/petkaantonov/bluebird)-ism that didn't get removed when we removed that dependency.

I've released v1.2.13 which should fix the problem, please let me know.

**Comment by Sk1Zy** [comment]
Works like a charm :) :+1:
